frontend/packages/ux-editor/src/components/config/componentSpecificContent/ComponentSpecificContent.tsx

```diff
@@ -1,44 +1,36 @@
 import React from 'react';
-
-import { ComponentTypes } from '../..';
-import type { IFormImageComponent } from '../../../types/global';
-import type { IGenericEditComponent } from '../componentConfig';
 import { ImageComponent } from './Image';
 import { ButtonComponent } from './Button';
 import { AddressComponent } from './Address';
 import { FileUploadComponent } from './FileUpload';
+import { SelectComponent } from './Select';
+import type { IGenericEditComponent } from '../componentConfig';
+import { ComponentTypes } from '../..';
+import { useText } from '../../../hooks';
+import type { IFormImageComponent } from '../../../types/global';
 
 export function ComponentSpecificContent({
   component,
-  handleComponentChange,
+  handleComponentChange
```

Comment:
I'm wondering if there is some syncing issue with our Prettier setup, does this line get changed back with `yarn lint --fix`?